src/Symfony/Component/Messenger/composer.json

```diff
@@ -21,10 +21,13 @@
     },
     "require-dev": {
         "psr/cache": "^1.0|^2.0|^3.0",
+        "symfony/cache": "^5.4|^6.0",
+        "symfony/clock": "^6.2",
         "symfony/console": "^5.4|^6.0",
         "symfony/dependency-injection": "^5.4|^6.0",
         "symfony/event-dispatcher": "^5.4|^6.0",
         "symfony/http-kernel": "^5.4|^6.0",
+        "symfony/lock": "^5.4|^6.0",
```

Comment:
messenger needs neither cache nor clock not lock, isn't it?

Reply:
leftover